core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -2151,16 +2151,16 @@ public boolean isTransactionRelevant(Transaction tx) throws ScriptException {
     }
 
     /**
-     * A transaction is mature if it is either a building coinbase tx that is as deep or deeper than the required coinbase depth, or a non-coinbase tx.
+     * Determine if a transaction is <i>mature</i>. A coinbase transaction is <i>mature</i> if it has been confirmed at least
+     * {@link NetworkParameters#getSpendableCoinbaseDepth()} times. On {@link BitcoinNetwork#MAINNET} this value is {@code 100}.
+     * For purposes of this method, non-coinbase transactions are also considered <i>mature</i>.
+     * @param tx The transaction to evaluate.
+     * @return {@code true} if it is a mature coinbase transaction or if it is not a coinbase transaction.
```

Comment:
Great JavaDoc! I'd begin `@params` and `@return` with small caps and not end on a sentence mark, because it's usually not a sentence. But that's not really important.

Reply:
Will fix.

Reply:
Fixed and force-pushed.